src/pages/not-found/index.jsx

```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+import Box from "@mui/material/Box";
+import { useTranslation } from "react-i18next";
+
+export default function Insights() {
+  const { t } = useTranslation();
+  return (
+    <center>
+      <Box
+        component="img"
+        sx={{
+          width: { xs: "100%", md: "60%" },
+          height: "auto",
+        }}
+        alt="404 not found"
+        src={require("../../assets/404.png")}
+      />
+      <Box
+        sx={{
+          typography: "h1",
+          fontSize: "2em",
+          marginTop: "-5rem",
```

Comment:
Using a negative margin is not recommended 😉 

![image](https://user-images.githubusercontent.com/45398769/174542202-db8f3a15-1df5-4b46-8e78-5d7e99f9c6d4.png)